.github/workflows/build.yaml

```diff
@@ -20,13 +23,13 @@ jobs:
 
       - name: Install Trivy
         run: |
-            curl -fsSL -o trivy.tar.gz https://github.com/aquasecurity/trivy/releases/download/v${{ env.TRIVY_VERSION }}/trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
-            tar -zxvf trivy.tar.gz
-            cp trivy /usr/local/bin/
+          curl -fsSL -o trivy.tar.gz https://github.com/aquasecurity/trivy/releases/download/v${{ env.TRIVY_VERSION }}/trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          tar -zxvf trivy.tar.gz
+          cp trivy /usr/local/bin/
 
-      - name: Generate trivy vuln report for nginx image
+      - name: Generate trivy vuln report for opa image
         run: |
-          trivy image --vuln-type os --ignore-unfixed -f json -o /tmp/nginx.1.21.6.json docker.io/library/nginx:1.21.6
+          trivy image --vuln-type os --ignore-unfixed -f json -o /tmp/opa.0.46.0.json docker.io/openpolicyagent/opa:0.46.0
```

Comment:
this is fine for now, but we'll need to add a trivy ignore list like copa in the future as there's no way to avoid false positives

Reply:
Created issue to add that: https://github.com/project-copacetic/copa-action/issues/38